src/database/seeders/20210726151520-mock-data-users.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert(
+      'Users',
+      [
+        { id: 1, createdAt: new Date(), updatedAt: new Date() },
+        { id: 2, createdAt: new Date(), updatedAt: new Date() },
+      ],
+      {}
+    );
+  },
+
+  down: async (queryInterface, Sequelize) => {
```

Comment:
Убрать неиспользуемый параметр
```suggestion
  down: async (queryInterface,) => {
```

Reply:
исправлен